src/Symfony/Component/Message/MessageBus.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Message;
+
+/**
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ * @author Matthias Noback <matthiasnoback@gmail.com>
+ */
+class MessageBus implements MessageBusInterface
```

Comment:
`MiddleawareMessageBus` :D

Reply:
if we took that the interface is called `MessageBusMiddlewareInterface` then it should be `MessageBusMiddleware` which is more accurate